react/modules/schema.ts

```diff
@@ -5,6 +5,7 @@ export interface Props {
   allowCookies?: boolean
   allowStyles?: boolean
   hidden?: boolean
+  [key: string]: any
```

Comment:
⛏This could be a `Record<string, any>`

Reply:
@victorhmp are you sure? 🤔 this is an index signature for the interface `Props`, for additional unknown props, not a general object type

Reply:
No it couldn't